luxjs/forms/form.js

```diff
@@ -94,7 +110,7 @@
                     'color': {element: 'input', type: 'color', editable: true, textBased: false},
                     'file': {element: 'input', type: 'file', editable: true, textBased: false},
                     'range': {element: 'input', type: 'range', editable: true, textBased: false},
-                    'select': {element: 'select', editable: true, textBased: false},
+                    'select': {element: 'select', editable: true, textBased: false, widget: formDefaults.select.widget.selectUI},
                     //  Pseudo-non-editables (containers)
```

Comment:
We need a way to override this ``widget`` which by default should be not specified.
We could move the ``supported`` object outside as a module ``value``, maybe called ``formElements``, so that it can be overwritten in the ``config`` method at application level.
```javascript
.value('formElements`, {...})
```

Then in our application (bmll) somewhere:
```javascript
.config(['formElements', function(formElements) {
    formElements.select.widget = 
}])
```